meson_options.txt

```diff
@@ -19,3 +19,4 @@ option('enable-element-restriction', type: 'boolean', value: false) # true to re
 option('restricted-elements', type: 'string', value: '')
 option('enable-nnfw', type: 'boolean', value: false)
 option('enable-nnfw-runtime', type: 'boolean', value: false)
+option('enable-cppfilter', type: 'boolean', value: false)
```

Comment:
How about adding this option with default true? Then we don't need to set build option in .spec and debian-rules.

Reply:
Yup.